### DISCUSSION TITLE
WebView / WebRender Anytype "client"

### DISCUSSION THREAD

**Discussion by bloodwiing** [anyproto/contributors#176] [Anytype apps]
I saw the topic #141 and the mentioned discussion in it, and do understand that the goal of Anytype is offline-first, so a web client complicates that.

My original need for a "webclient" was not to edit notes, but rather have an online HTML document of them that I could view from anywhere without the need to log into Anytype.

Like say a Space for typing out blogs or articles and those written out pages would appear on a webserver with the same look and non-mutable function.

So my thought is: could (or would) there be an anytype-heart client that waits for updates in sync, receives them, renders out an HTML page for the changed pages and serve them as a webserver?

**Comment by fuksman** [comment]
Hi @bloodwiing!
Sounds like you're talking about publish-to-web feature: https://github.com/orgs/anyproto/projects/1?pane=issue&itemId=29227800
We have it in our roadmap, but without particular estimates yet.

**Reply by bloodwiing** [reply]
Oh I misunderstood what "publish" meant then. I assumed it was just an HTML or web export

From what it looks like, this seems to actually be planned how I imagined it. Just hoping to see the feature sooner now haha
